Makefile

```
@@ -10,13 +10,12 @@ help:
 .PHONY: bootstrap
 bootstrap: ## Install build dependencies
 	mkdir -p logs screenshots
-	pip install -r requirements_for_test.txt
+	uv pip install -r requirements_for_test.txt
 
 .PHONY: freeze-requirements
 freeze-requirements: ## create static requirements_for_test.txt
-	pip install --upgrade pip-tools
 	python -c "from notifications_utils.version_tools import copy_config; copy_config()"
-	pip-compile requirements_for_test.in
+	uv compile requirements_for_test.in -o requirements_for_test.txt
```

```suggestion
	uv pip compile requirements_for_test.in -o requirements_for_test.txt
```
(at least on my local machine `uv compile` gives `error: unrecognized subcommand 'compile'`

---

oops yes you're right